.pre-commit-config.yaml

```diff
@@ -1,38 +1,38 @@
-repos:
--   repo: https://github.com/pre-commit/pre-commit-hooks
-    rev: v2.3.0
-    hooks:
-    -   id: check-builtin-literals
-        args: ['--no-allow-dict-kwargs']
-        exclude: bumpity.py$
-    -   id: check-docstring-first
-    -   id: debug-statements
-        exclude: bumpity.py$
-    -   id: double-quote-string-fixer
-    -   id: end-of-file-fixer
-    -   id: name-tests-test
-    -   id: flake8
-        args: ['--max-line-length', '100']
-        exclude: ^test_data/|bumpity.py$
-    -   id: trailing-whitespace
--   repo: https://github.com/asottile/reorder_python_imports
-    rev: v1.6.1
-    hooks:
-    -   id: reorder-python-imports
-        language_version: python3
-        exclude: bumpity.py$
--   repo: https://github.com/asottile/add-trailing-comma
-    rev: v1.4.1
-    hooks:
-    -   id: add-trailing-comma
--   repo: https://github.com/pre-commit/mirrors-autopep8
-    rev: v1.4.4
-    hooks:
-    -   id: autopep8
--   repo: local
-    hooks:
-    -   id: detect-secrets
-        name: Detect secrets
-        language: python
-        entry: detect-secrets-hook
-        args: ['--baseline', '.secrets.baseline']
+repos:

```

Comment:
Did anything change here?

Reply:
Oups, I think this could be an error between WSL2 (Linux) - Git (Windows), I will revert this no-changes now.